client/components/admin.jsx

```diff
@@ -0,0 +1,339 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable no-shadow */
+/* eslint-disable react/state-in-constructor */
+import Alert from 'react-s-alert';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import propTypes from 'prop-types';
+import moment from 'moment';
+import {
+  getMovies, searchImdb, orderStock, removeMovie, addStock,
+} from '../redux/movies/actions';
+import { getUsers, toggleAdmin } from '../redux/users/actions';
+import { adminPreviousOrders } from '../redux/cart/actions';
+import { adminInventoryFilter } from '../utilities';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Admin extends Component {
+  state = {
+    searchInput: '',
+    stockSearch: '',
+  }
+
+  async componentDidMount() {
```

Comment:
async no await??

Reply:
1. This will be performance cost-ly! 
2. componentDidMount should **NOT** be async.